tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot

```diff
@@ -101,7 +101,7 @@ Create Kafka Stream Instance
   END
   Input Text    xpath=//input[@id='form-instance-name']    ${stream_name}
   Click Element    xpath=//div[text()='${cloud_provider}']
-  Select From List By Value    id:cloud-region-select   ${stream_region}
+  Select From List By Value    id:form-cloud-region-option   ${stream_region}
```

Comment:
Code looks fine . @bdattoma @lugi0 just a thought could we add multiple locator with "or" using conatin or something which will be dynamic so that in case wither id gets changed (edge case) or  anything gets changed in future it will not impact our test suites?

Reply:
Mh, that would work only in the case where the id (or other locator) is changed in such a way that part of the original is still included. I'm not sure it would solve most of these kinds of issues we have encountered?

Reply:
In practice, it can be done. The only "issue" I can see doing this is the code will contains outdated pieces and we wouldn't know. It is just a thought

In general, it could be helpful when we must write some code to be compatible with two different platform version and no common locators can be found.

Reply:
> Mh, that would work only in the case where the id (or other locator) is changed in such a way that part of the original is still included. I'm not sure it would solve most of these kinds of issues we have encountered?

hmm. May be union(|) will be helpful in this case

Reply:
> In practice, it can be done. The only "issue" I can see doing this is the code will contains outdated pieces and we wouldn't know. It is just a thought
> 
> In general, it could be helpful when we must write some code to be compatible with two different platform version and no common locators can be found.

Totally Agree.There will be chance of having outdated xpath. yes, it would be helpful for platform compatibility..In some case it will help us not rewrite the whole block of code using If Else block

Reply:
agree with that. Let's have a dedicated discussion on that when we have a case we want to solve